src/main_ghostty.zig

```diff
@@ -123,6 +123,10 @@ fn logFn(
     const level_txt = comptime level.asText();
     const prefix = if (scope == .default) ": " else "(" ++ @tagName(scope) ++ "): ";
 
+    if (!state.log_enabled or @intFromEnum(level) > @intFromEnum(state.log_level)) {
```

Comment:
IMO comparing log levels via their integer representation is bad style. Write an explicit switch case.

Reply:
That was the most recommended solution for changing log levels at runtime and is how log.zig itself does it.  I suppose I could change it to a simple equality test.  Only log if the message is an error or if verbose is true.

Reply:
Since I've removed the `-q/--quiet` option, there's no longer a need to compare the levels.  It's just...

```
if (!state.verbose_logging and level != .err) {
    return;
}
```